src/components/EntryCards.js

```diff
@@ -35,7 +35,7 @@ export default function EntryCards(props) {
   const [showModal, setshowModal] = useState(false);
   const [isPublic, setVisibility] = useState(props.content.privacy === 'PUBLIC');
   const [authorName, setAuthorName] = useState('');
-  const [isEditable, setEditable] = useState(false)
+  const [isEditable, setEditable] = useState(true);
```

Comment:
Should we set this to true by default?

Reply:
Set to true for development, will change to false later